src/plugins/intel_cpu/src/utils/jit_kernel.hpp

```diff
@@ -788,17 +787,19 @@ void jit_kernel::store(const variable<DstT> & dst, const variable<SrcT[N]> & src
     const std::vector<size_t> pool_vec_idxs(_free_rmmregs.begin(), _free_rmmregs.end());
     const std::vector<size_t> pool_gpr_idxs(_free_x64regs.begin(), _free_x64regs.end());
 
-    _store_emitter.emit_code(
+    const auto src_prc = internal::type2precision<src_type>();
+    const auto dst_prc = internal::type2precision<dst_type>();
+
+    const auto key = store_emitter_params(src_prc, dst_prc, length).hash();
+    if (!_emitters[key]) {
+        _emitters[store_emitter_params(src_prc, dst_prc, length).hash()].reset(
```

Comment:
"store_emitter_params(src_prc, dst_prc, length).hash()" is already computed above - you can reuse it.